build-system/babel-config/unminified-config.js

```diff
@@ -30,6 +30,7 @@ function getUnminifiedConfig(buildFor = 'preact') {
       loose: true,
       targets,
       shippedProposals: true,
+      exclude: argv.esm || argv.sxg ? ['@babel/plugin-transform-for-of'] : [],
```

Comment:
nit: I think we can skip this line for unminified builds right. is there a reason for it?

Reply:
Consistency between builds.